docker/s6/services/.s6-svscan/clear_cache.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+if [ -z "${RASTER_BASE_DIR}" ]; then
+    echo "Failed to clear the lazyraster cache: RASTER_BASE_DIR not set"
+    return 1
+fi
+
+echo "Trying to clear the lazyraster file cache..."
+
+rm -rf ${RASTER_BASE_DIR}
```

Comment:
This really should be more careful to remove only PDF files and empty directories. It's kind of a machine gun pointed at your foot right now.